experimental/dds/tree2/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -186,12 +195,16 @@ function invertMark<TNodeChange>(
 						),
 					];
 				}
+
+				const deleteMark: Delete<TNodeChange> = {
+					type: "Delete",
+					count: mark.count,
+					id: mark.cellId.localId,
+					detachIdOverride: mark.cellId,
```

Comment:
Should this CellID (when specified) not be used when inverting such a delete? (Line 154 at the time of this comment)
This line (203) is fine, I just can't put comments on line 154.

Reply:
Good point

Reply:
There is an existing "delete => revive" test. We should add a variant of it that has the override specificed on it.